auslib/admin/views/releases.py

```diff
@@ -303,6 +303,39 @@ def _delete(self, release, changed_by, transaction):
         return Response(status=200)
 
 
+class ReleaseReadOnlyView(AdminView):
+    """/releases/:release/read_only"""
+
+    def get(self, release):
+        releases = dbo.releases.getReleases(name=release, limit=1)
+        is_release_read_only = dbo.releases.isReadOnly(name=release, limit=1)
```

Comment:
I don't think you need to do both of these lookups. If you just do the isReadOnly one inside of a try/except block, you can figure out whether or not it exists by catching an exception.